Src/Plugins/Security/SEC101_038.PostgreSqlCredentialsValidator.cs

```diff
@@ -68,7 +71,8 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(Dictionary<
 
             // Username must be in the form <username>@<hostname> to communicate with Azure.
             // If the username does not contain a host name, we can't connect.
-            if (hostValue.Contains(AzureHost) && !idValue.Contains("@"))
+            if (AzureHosts.Any(azHosts => hostValue.IndexOf(azHosts, StringComparison.OrdinalIgnoreCase) != -1) &&
```

Comment:
Again, `.Contains()` might be a bit more readable here.

Reply:
Contains does not have stringcomparison :)

Reply:
the version  that contain 'contains' with StringComparison is net5, which does not apply here.